src/main/java/nextstep/ladder/vo/DirectionType.java

```diff
@@ -0,0 +1,27 @@
+package nextstep.ladder.vo;
+
+import nextstep.ladder.domain.line.Lines;
+
+public enum DirectionType {
+    DOWN, LEFT, RIGHT;
+
+    public static DirectionType checkDirection(int row, int col, Lines lines) {
+        if (lines.isLeftDirection(row, col)) {
+            return LEFT;
+        }
+        if (lines.isRightDirection(row, col)) {
+            return RIGHT;
+        }
+        return DOWN;
+    }
+
+    public int moveRow() {
+        if (this.equals(LEFT)) {
+            return -1;
+        }
+        if (this.equals(RIGHT)) {
+            return 1;
+        }
+        return 0;
+    }
+}
```

Comment:
```java
@FunctionalInterface
public interface MoveStrategy {
    int move();
}
```

전략을 추가하고 람다로 구체화하여 enum을 활용하는 방법도 있습니당 😄 

```suggestion
public enum DirectionType {
    DOWN(() -> 0),
    LEFT(() -> -1),
    RIGHT(() -> 1)
    ;

    private final MoveStrategy moveStrategy;

    DirectionType(MoveStrategy moveStrategy) {
        this.moveStrategy = moveStrategy;
    }

    public static DirectionType checkDirection(int row, int col, Lines lines) {
        if (lines.isLeftDirection(row, col)) {
            return LEFT;
        }
        if (lines.isRightDirection(row, col)) {
            return RIGHT;
        }
        return DOWN;
    }

    public int moveRow() {
        return moveStrategy.move();
    }
}
}
```

Reply:
오 이런방법이 있군요 적용하겠습니다~